FAQ/en.md

```diff
@@ -0,0 +1,73 @@
+# Frequently Asked Questions
+
+## General
+
+### What is Quaver?
+
+Quaver is a community-driven rhythm game, aiming to bring VSGRs to the next level. The game is open to players of all skill levels, casual and seasoned.
+
+### How is Quaver different from other games?
+
+The game is fully open-source, allowing anyone to help and improve the game. Furthermore, we are aiming to provide a new VSGR experience by bringing competitive concepts, such as competitive matchmaking.
+
+### How far in development is Quaver?
+
+Quaver is still in early development and is currently in closed alpha. In order to use online features of Quaver, you need to sign up for the alpha.
+
+### What platforms will Quaver be released on?
+
+Quaver will be released on Windows, Mac and Linux.
+
+### Where can I sign up for the Quaver Alpha?
+
+Registrations for the closed alpha occur in rounds. Each round of sign-ups are announced before they open.
+
+### Can I play without being part of the Quaver Alpha?
+
+Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features including online rankings, map uploading, and other community-based features such as in-game chat.
+
+### Does this mean I can’t download maps from the Quaver website?
+
+You cannot download maps from the Quaver website, however you can still download packs via a [mirror](https://rhythmgamers.net/pack/).
+
+### Is there an estimate for the public release date?
+
+We do not have an estimate for the public release date yet.
+
+
+## Gameplay
+
+### What game modes does Quaver support?
+
+Quaver currently only supports 4 keys (4k) and 7 keys (7k).
+
+### Can I import maps from other games?
+
+Yes, Quaver currently supports .osz and .sm files. To import them, drag the files into the game window.
+
+### I have a lot of maps in an external game, can I import them all at once?
+
+It is possible to import all of your maps at once by linking the game’s “.db” file inside your “quaver.cfg” file, which can be located inside your Quaver folder.
+
+### Can I import my skin from another game?
+
+The game doesn't support skin conversions out of the box, however you can download a .osk skin converter [here](https://rhythmgamers.net/QBC/) along with a [tutorial](https://www.youtube.com/watch?v=pWeLbx48NVI).
+
+### I found a bug, where can I report it?
+
+Any bug reports should be posted on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Make sure to post anything you find, but please check if the issue has already been reported to avoid duplication.
+
+### I have a feature request, where can I post it?
+
+Feature requests belong, just like bug reports, on our [GitHub issues](https://github.com/Quaver/Quaver/issues). Simply choose “Feature Request” when creating a new issue!
```

Comment:
Similar to bug reports, feature requests belong on our....

Reply:
done